app/views/content_items/travel_advice.html+print.erb

```diff
@@ -45,10 +52,17 @@
 
     <%= render 'govuk_publishing_components/components/print_link', {
       data_attributes: {
+        module: "ga4-event-tracker",
         "track-category": "printButton",
         "track-action": "clicked",
         "track-label": t("components.print_link.text"),
-        module: "print-link"
+        ga4: {
+          event_name: 'print_page',
+          type: 'Print this page',
+          index: 1,
+          index_total: 1,
```

Comment:
Just to check: would this need to be `index: 2` and `index_total`: 2?

Reply:
Yes, great spot, thanks, have updated.